src/cli/filter_arg.zig

```diff
@@ -0,0 +1,195 @@
+const std = @import("std");
+const root = @import("root");
+const bun = root.bun;
+const string = bun.string;
+const Output = bun.Output;
+const Global = bun.Global;
+const strings = bun.strings;
+const json_parser = bun.JSON;
+const Glob = @import("../glob.zig");
+
+const Package = @import("../install/lockfile.zig").Package;
+
+fn findWorkspaceMembers(allocator: std.mem.Allocator, log: *bun.logger.Log, workspace_map: *Package.WorkspaceMap, workdir_: []const u8) !void {
```

Comment:
In large directory trees, this approach takes seconds to show any output. We cannot ship it without being an iterator.

Reply:
understood, will get on that 👍